src/components/Messages/Message.tsx

```diff
@@ -77,7 +77,7 @@ const Message: FC<MessageProps> = ({ conversationKey }) => {
                 hasMore={hasMore}
                 missingXmtpAuth={missingXmtpAuth ?? false}
               />
-              <Composer sendMessage={sendMessage} />
+              <Composer sendMessage={sendMessage} missingXmtpAuth={missingXmtpAuth ?? false} />
```

Comment:
I would name this something more generic like "disabled", so that the component is more reusable

Reply:
Dig it! Updated.